File: 58.go
func lengthOfLastWord(s string) int {
	i := len(s) - 1
	ans := 0
	for ; i >= 0; i-- {
		if s[i] != ' ' {
			break
		}
	}
	for ; i >= 0 && s[i] != ' '; i-- {
		ans++
	}
	return ans
}

File: 58.java
class Solution {
    public int lengthOfLastWord(String s) {
        int ans = 0, i;
        for(i = s.length()-1;i >= 0;i --) {
            if(s.charAt(i) != ' ') break;
        }
        while(i >= 0 && s.charAt(i) != ' ') {
            i --;
            ans ++;
        }
        return ans;
    }
}